libraries/AC_AutoTune/AC_AutoTune_FreqResp.cpp

```diff
@@ -258,14 +262,14 @@ void AC_AutoTune_FreqResp::push_to_meas_buffer(uint16_t count, float amplitude,
     sample.curr_count = count;
     sample.amplitude = amplitude;
     sample.time_ms = time_ms;
-    meas_peak_info_buffer.push(sample);
+    meas_peak_info_buffer->push(sample);
```

Comment:
There is no `nullptr` check here. Maybe there is one before this function is called?

Reply:
done. thanks.